interfaces/migrate/src/main/java/migrate/interfaces/JavaLib.java

```diff
@@ -0,0 +1,34 @@
+package migrate.interfaces;
+
+
+public class JavaLib implements Lib {
+    public String organization;
+    public String name;
```

Comment:
Why make this `public`? You already have getters.
I think you can either:
 - Keep the getters and make these attributes `private final String`
 - Or drop the getters and make these attributes `public final String`
This is clearly a data class, so immutable is better :)

Reply:
ok thanks.